examples/basic_usage/outputs.tf

```diff
@@ -1,14 +1,14 @@
-output "vpc_id" {
-  value       = aws_vpc.the_vpc.id
-  description = "The VPC ID"
+output "vpc" {
+  value       = aws_vpc.the_vpc
+  description = "The VPC"
```

Comment:
I think the descriptions in this file should also end in periods.

Reply:
See commit cc28900.